lesson_03/quiz/lesson_03_quiz/src/test/resources/application.yml

```diff
@@ -12,9 +12,6 @@ quiz:
     anthonymays:
      - $2y$10$8eHSzy3aCu4Ry3LzO9nWCeGpofSxsNVbnF.wCfn3ZADwQ6MEtN/KK
      - $2y$10$55EXRjF26JIgebtoH800ZOJecfefvMgHicuxf/rwTENuxiUaFQcNe
-    moibrahi7:
```

Comment:
Your change would delete the answers for Mohamed and Javy. Please add their answers back.